javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+var bookArray = [
+  {
+    isbn: 123,
+    title: 'css',
+    author: 'john'
+  },
+  {
+    isbn: 456,
+    title: 'html',
+    author: 'dave'
+  },
+  {
+    isbn: 789,
+    title: 'js',
+    author: 'bob'
+  }
+];
+
+console.log(bookArray);
+console.log(typeof bookArray);
+
+var BookString = JSON.stringify(bookArray);
```

Comment:
remember you want to avoid variable names with Capital letters at the beginning.